### PR TITLE
Update webhooks.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

**Comment by icarusdust** [comment]
Fixes #393

File: content/configuration/webhooks.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ You can find your app ID in the browser URL after `app/` when you open the app o
 
 All received webhooks are visible in the Codemagic UI when navigating to your application and selecting the **Webhooks** tab.
 
+After configuring **Webhooks**, automatic build triggering can be set up as explained [here](https://docs.codemagic.io/yaml/yaml-getting-started/#triggering)
+
```
Comment (nevercode-kevin) [inline content/configuration/webhooks.md:19]: I think that it needs to be two links. 
- One for how to set up triggers in Workflow Editor
- One for how to set it up in codemagic.yaml

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Comment by icarusdust** [comment]
Added two separate links.

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**